### PR TITLE
Bugfix #182315539 - Add organism in payload field - for v6 (used in production)

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
A new iteration of the collection so that we get metadata suggestions. Notice that the suggesters themselves *were* returning suggestions correctly, but we couldn’t filter them out in the back end, and this caused no metadata suggestions to be sent to the search box at all. Unfortunately there’s no way around this bug without significant changes in the suggesters.

A summary:
- The main change is to add `ctw_organism` in the payload, so we can filter by species in the back end.†
- Centralise the version of `scxa-analytics` in a sourced file. This is better than having it scattered among over ten shell scripts. We should probably do this with the other collections at some point.
- Move the block that tests that the suggesters work to the tests.
- Removed the child suggester. We don’t have children in our documents and we don’t want them. All results will be empty.
- Disable highlighting until our front end components show suggestions with fancy HTML formatting.

† By default, the back end uses one or more species as [the context filter in the Solr suggester](https://solr.apache.org/guide/7_1/suggester.html#context-filtering). However [there’s no contextField defined](https://github.com/ebi-gene-expression-group/index-scxa/pull/28), which causes no suggestions to be returned when the `suggest.cfq` request parameter is populated. 

Even more problematic is that if we add `ctw_organism` as the context field, it still dosn’t work because for some reason Solr doesn’t parse space-separated fields properly. [According to the source code it uses a StandardTokenizer](https://github.com/apache/solr/blob/c99af207c761ec34812ef1cc3054eb2804b7448b/solr/core/src/java/org/apache/solr/spelling/suggest/SolrSuggester.java#L110) which should work, but either the request parameter is sent as a raw string or the source code for Solr 7.1 is buggy.

For the time being we can do the context filtering on the back end by adding the species (i.e. `ctw_organism` as a payload field).

### CODE REVIEW

**Comment by pcm32** [comment]
Could you please rebase this on the branch being used in https://github.com/ebi-gene-expression-group/index-scxa/pull/35? There won't be a new data release without 8.7 I suspect, as we are very close to moving there completely. Thanks!

**Comment by pcm32** [comment]
I have fixed tests on that branch now.

**Comment by alfonsomunozpomer** [comment]
Superseded by https://github.com/ebi-gene-expression-group/index-scxa/pull/37.

**Comment by alfonsomunozpomer** [comment]
> Could you please rebase this on the branch being used in #35? There won't be a new data release without 8.7 I suspect, as we are very close to moving there completely. Thanks!

[Done!](37)

File: run_tests_in_containers.sh [hunk 1]
```diff
@@ -10,6 +10,6 @@ docker run --net mynet --name my_solr -v $( pwd )/tests:/usr/local/tests -v $(pw
 docker build -t test/index-scxa-module .
 sleep 20
 
-docker exec -it --user=solr my_solr bin/solr create_collection -c scxa-analytics-v6
+docker exec -it --user=solr my_solr bin/solr create_collection -c scxa-analytics-v7
```
Comment (pcm32) [inline run_tests_in_containers.sh]: Needs to be reverted to v6

**Comment by alfonsomunozpomer** [comment]
See linked issue, we don’t want to merge the branch, just run it once.

**Pull request closed without merging**